packages/vite-plugin/README.md

```diff
@@ -18,8 +15,7 @@ A Vite plugin that integrates with Netlify's platform features.
 | Environment Variables  | ✅ Yes    |
 | Image CDN              | ✅ Yes    |
 
-> Note: Missing features will be added incrementally. This module is **not** intended to be a full replacement for the
-> Netlify CLI.
+> This module is **not** intended to be a full replacement for the Netlify CLI.
```

Comment:
I feel like this should be either removed or clarified. It's very vague. When should you use `netlify dev`? Or should we just say it's a replacement for `netlify dev`, not a replacement for the full Netlify CLI?

Reply:
Will follow up in a later PR